src/components/SelectInput.tsx

```diff
@@ -28,9 +33,13 @@ export const SelectInput: React.FunctionComponent<IFormtronControl> = ({
               ? async () => schema.options.map((o: string) => ({ value: o, label: o }))
               : async (search: string) => [{ value: search, label: search }];
         return (
-          <Flex width="100%">
-            <Box flex="1" as="label" htmlFor={id}>
-              {schema.title}
+          <Flex width="100%" alignItems="center">
+            <Box flex="1">
+              <Messages path={path}>
```

Comment:
Seems to be a candidate for a separate component.

Reply:
Eh, thing is that would be complecting Layout concerns (Flex, Box) with Function concerns (tooltip target area).

Until the very last iteration, `<Messages>` actually wrapped a completely different set of components, not just the `<Label>`.

What would you even _*call*_ it? 🤷‍♂️

Reply:
`Title` perhaps?

Reply:
Anyway, don't feel obliged to change it. I am fine with it as is.